web/index.js

```diff
@@ -138,23 +138,17 @@ function getEventsByType(type) {
  */
 function init() {
   renderUI([], _user);
-
   const promises = [getEventsByType('comment'), getEventsByType('event')];
-
   Promise.all(promises);
 }
 
-document
-  .getElementById(botName)
+document.getElementById(botName)
```

Comment:
i wonder if we should extract these into a different file for constants for listeners and their events and then pass them all into a function rather than needing a bunch of them like this 

something like this
['click','ontouchstart'].forEach( evt => 
    element.addEventListener(evt, dosomething, false)
);